src/mmw/apps/modeling/views.py

```diff
@@ -312,6 +312,54 @@ def start_analyze(request, format=None):
     )
 
 
+@decorators.api_view(['POST'])
+@decorators.permission_classes((AllowAny, ))
+def start_analyze_animals(request, format=None):
+    user = request.user if request.user.is_authenticated() else None
+    created = now()
+    area_of_interest = request.POST['area_of_interest']
+    job = Job.objects.create(created_at=created, result='', error='',
+                             traceback='', user=user, status='started')
+
+    chain = (tasks.analyze_animals.s(area_of_interest) |
+             save_job_result.s(job.id, area_of_interest))
+    task_list = chain.apply_async(link_error=save_job_error.s(job.id))
+
+    job.uuid = task_list.id
+    job.save()
+
+    return Response(
+        {
+            'job': task_list.id,
+            'status': 'started',
+        }
+    )
+
+
+@decorators.api_view(['POST'])
+@decorators.permission_classes((AllowAny, ))
+def start_analyze_pointsource(request, format=None):
+    user = request.user if request.user.is_authenticated() else None
+    created = now()
+    area_of_interest = request.POST['area_of_interest']
+    job = Job.objects.create(created_at=created, result='', error='',
+                             traceback='', user=user, status='started')
+
+    chain = (tasks.analyze_pointsource.s(area_of_interest) |
+             save_job_result.s(job.id, area_of_interest))
+    task_list = chain.apply_async(link_error=save_job_error.s(job.id))
```

Comment:
There's a lot of rhyming in this pattern here. We should perhaps one day try to refactor this into something reusable. A lot of these variables are interdependent though, so it would be non-trivial to disentangle them.